rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -36,7 +34,7 @@
     },
     "barcode": {
       "title": "Barcode",
-      "description": "Barcode of the item.",
+      "description": "Barcode of the item. This field is optional",
```

Comment:
See with Igor. I think it should be finished by a ".".
Maybe just "(optional)" string is enough.

Reply:
@zannkukai is right, trailing dot. But more than that, is this description actually useful for the end user? I think the title is enough. And "required or optional" should be visually displayed (red * and validation message).

So I would delete the description field. See [JSON schema guidelines, section description](https://github.com/rero/developer-resources/blob/master/guidelines/json-schemas.md#description).